code/modules/security_levels/security_levels.dm

```diff
@@ -45,6 +49,18 @@ GLOBAL_VAR_INIT(security_level, SEC_LEVEL_GREEN)
 						modTimer = 0.5
 				else
 					minor_announce(CONFIG_GET(string/alert_red_downto), "Attention! Code red!")
+          
+			if(SEC_LEVEL_GAMMA)
+				minor_announce(CONFIG_GET(string/alert_gamma), "Attention! Gamma security level activated!", TRUE)
+				SEND_SOUND(world, 'sound/misc/gamma_alert.ogg')
+				if(GLOB.security_level == SEC_LEVEL_GREEN)
+					modTimer = 0.25
+				else
+					modTimer = 0.5
```

Comment:
Switching from gamma to red (or the other way around) will halve the shuttle timer each time, and it can never be reset.

Reply:
going from gamma to red should double the timer now.

Reply:
lmao

Reply:
NO